consensus/quorum/one-node-one-vote.go

```diff
@@ -0,0 +1,64 @@
+package quorum
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/harmony-one/bls/ffi/go/bls"
+	common2 "github.com/harmony-one/harmony/internal/common"
+	"github.com/harmony-one/harmony/staking/effective"
+)
+
+type uniformVoteWeight struct {
+	SignatureReader
+}
+
+// Policy ..
+func (v *uniformVoteWeight) Policy() Policy {
+	return SuperMajorityVote
+}
+
+// IsQuorumAchieved ..
+func (v *uniformVoteWeight) IsQuorumAchieved(p Phase) bool {
+	return v.SignatoriesCount(p) >= v.QuorumThreshold()
```

Comment:
Shall we just use Signers instead of Signatories? for simplicity and consistency with other places.

Reply:
Done